e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -506,6 +506,7 @@ az keyvault secret set --vault-name $keyVaultName --name "DPS-GLOBALDEVICEENDPOI
 # Below Environment variables are only used in Java
```

Comment:
Pass the WebRegion as optional parameter so that it helps when we create this resource?

Reply:
YAGNI?